events_ingress/src/main/java/com/adobe/event/publish/api/PublishApi.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.event.publish.api;
+
+import com.adobe.event.publish.model.CloudEvent;
+import feign.Headers;
+import feign.RequestLine;
+
+@Headers("Accept: application/json")
+public interface PublishApi {
+
+  String DEFAULT_URL = "https://eventsingress.adobe.io";
+
+  /**
+   * publish a Cloud Event
+   *
+   * @param body your CloudEvent Input Model
+   */
+  @RequestLine("POST ")
```

Comment:
there is a space here. not sure if its intentional "POST"

Reply:
yes extra space was not harmful but not needed either